### PR TITLE
Feat/993 pagination params

### PR DESCRIPTION
**Pull request by EliNoden**
## Pourquoi

La route GET /meetings ne prend pas en compte les params page et page_size . On veut mettre en place la pagination côté back.

## Quoi

Ajout de `page` et `page_size` dans les paramètres des différentes méthodes. 
Ajout de tests pour valider le bon fonctionnement des params et gestion des edge cases. 

## Comment tester

ETQDev je peux lancer les tests et m'assurer qu'ils passent tous

## Checklist
- [x] J’ai lancé les tests
- [x] J’ai lancé le lint
- [x] J’ai mis à jour la doc/README si nécessaire
- [x] Pas de secrets/credentials ajoutés

### CODE REVIEW

File: mcr-gateway/mcr_gateway/app/services/meeting_service.py [hunk 1]
```diff
@@ -277,22 +277,29 @@ async def stop_meeting_capture_service(
...
     """
     try:
-        params = {}
+        params: Dict[str, Union[str, int]] = {}
 
         if search:
             params["search"] = search
 
+        params["page"] = page
+        params["page_size"] = page_size
```
[the remark this answers is not in the document]
Comment (leogermain09) [reply mcr-gateway/mcr_gateway/app/services/meeting_service.py]: Modifié

File: mcr-core/mcr_meeting/app/db/meeting_repository.py [hunk 2]
```diff
@@ -73,17 +73,24 @@ def update_meeting(updated_meeting: Meeting) -> Meeting:
...
+    user_id: int, search: Optional[str], page: int, page_size: int
+) -> List[Meeting]:
     """
     Récupère une liste de réunions filtrées depuis la base de données.
 
     Args:
         user_id (int): ID de l'utilisateur.
         search (str): Terme de recherche optionnel pour filtrer les réunions.
+        page (int): Numéro de page (défaut: 1).
+        page_size (int): Nombre d'éléments par page (défaut: 10).
```
Comment (ThomasMetzger) [inline mcr-core/mcr_meeting/app/db/meeting_repository.py]: ⭐ ⭐ Il faut juste supprimer les (défaut: ...) dans la docstring

File: mcr-core/tests/orchestrators/test_meeting_repository.py [hunk 3]
```diff
@@ -46,8 +48,120 @@ def test_get_meetings_with_search_filters_deleted():
...
     assert deleted.id not in {m.id for m in results}
 
     assert {m.id for m in results} == {active1.id, active2.id}
+
+
+def test_get_meetings_pagination_defaults():
+    user = UserFactory.create()
+
+    for _ in range(15):
+        MeetingFactory.create(owner=user, status=MeetingStatus.IMPORT_PENDING)
```
Comment (ThomasMetzger) [inline mcr-core/tests/orchestrators/test_meeting_repository.py]: ⭐ Tu peux utiliser MeetingFactory.createBatch pour ce use case ;)

File: mcr-core/tests/orchestrators/test_meeting_repository.py [hunk 4]
```diff
@@ -46,8 +48,120 @@ def test_get_meetings_with_search_filters_deleted():
...
+    )
+
+    assert len(results) == 5
+
+
+def test_get_meetings_pagination_custom_page_size():
+    user = UserFactory.create()
+
+    for _ in range(15):
+        MeetingFactory.create(owner=user, status=MeetingStatus.IMPORT_PENDING)
```
Comment (ThomasMetzger) [inline mcr-core/tests/orchestrators/test_meeting_repository.py]: Et ici :)

**Comment by MorinAlexandre** [comment]
Commentaire fait en live:
Ne pas bloquer à 10 le page_size. Nous avons aucune contrainte de notre côté à retourner moins donc ajoutons seulement le min = 1
